rfc/003-account-hierarchy.md

```diff
@@ -0,0 +1,37 @@
+# RFC for openMFP Account Hierarchy
+
+## Context and Problem Statement
+
+This document provides an outline of how and what should be achieved with an account hierarchy model within openMFP. An account will have the ability to benefit from other accounts that it is a child of, e.g., by inheriting extensions provided by its parents. This will allow for easier distribution of central extensions and give more structure to the entire account setup.
```

Comment:
```suggestion
This document provides an outline of how and what should be achieved with an account hierarchy model within OpenMFP. An account will have the ability to benefit from other accounts that it is a child of, e.g., by inheriting extensions provided by its parents. This will allow for easier distribution of central extensions and give more structure to the entire account setup.
```